### PR TITLE
Use TextBlock instead of png in PullToRefresh

### PR DESCRIPTION
**Pull request by HO-COOH**
##  Pull Request Type
[ ] Bugfix <!—Please add link to related Issue below -->  
[ ] New Sample  
[x] Other - Please describe:   

## Issue
 #  

## Proposed Changes
Minor refactoring

## Screenshot/GIF (if applicable):
I managed to achive pretty similar results
image

### CODE REVIEW

File: SampleGallery/Samples/SDK 14393/PullToAnimate/PullToAnimate.xaml [hunk 1]
```diff
@@ -27,11 +27,23 @@
               RelativePanel.AlignRightWithPanel="True"
               RelativePanel.AlignBottomWithPanel="True">
             <Image x:Name="WeatherInfoImage" Source="ms-appx:///Assets/Other/WeatherInfo.png" HorizontalAlignment="Center" VerticalAlignment="Top" Width="383" Height="351" Margin="0,0,0,0"/>
+
         </Grid>
         <Grid x:Name="ToDoLayer" Width="500" Height="145" Margin="0,0,0,0"
               RelativePanel.AlignHorizontalCenterWithPanel="True"
               RelativePanel.AlignVerticalCenterWithPanel="True">
-            <Image x:Name="Appointment" Source="ms-appx:///Assets/Other/Appointment.png" HorizontalAlignment="Left" VerticalAlignment="Bottom" Width="500" Height="145" Margin="0,0,0,0"/>
+            <!--<Image x:Name="Appointment" Source="ms-appx:///Assets/Other/Appointment.png" HorizontalAlignment="Left" VerticalAlignment="Bottom" Width="500" Height="145" Margin="0,0,0,0"/>-->
```
Comment (jeffstall) [inline SampleGallery/Samples/SDK 14393/PullToAnimate/PullToAnimate.xaml:35]: Can you remove the old line complete?

**Review by jeffstall — CHANGES_REQUESTED** [review]
Looks good. Just remove the old line too.

File: SampleGallery/Samples/SDK 14393/PullToAnimate/PullToAnimate.xaml (hunk 1 above)
Comment (HO-COOH) [reply SampleGallery/Samples/SDK 14393/PullToAnimate/PullToAnimate.xaml]: > Can you remove the old line completely?

Done

**Review by jeffstall — APPROVED** [review]
Thanks!

**Comment by jeffstall** [comment]
We're archiving this repo and have moved everything to https://github.com/microsoft/WindowsAppSDK-Samples/tree/main/Samples/SceneGraph instead.

**Pull request closed without merging**